### ISSUE TITLE
Setup page redirect loop

### ISSUE THREAD

**Issue by Nezteb** [LinkStackOrg/linkstack-docker#55]
After removing/disabling any mounts/volumes ([because they cause the container to crash](https://github.com/JulianPrieber/llc-docker/issues/44#issuecomment-1500981900)), I'm able to get to the LittleLink Custom setup page, but I'm unable to complete it because it eventually sends me back to the beginning.

Video demo:

https://user-images.githubusercontent.com/3588798/230744015-d8d08cd2-3120-4dbd-965f-78a33e37572b.mov

I have `HTTP_SERVER_NAME` and `HTTPS_SERVER_NAME` both set to the same URL: `links.nezteb.net` (I've since taken the service down since I couldn't get it working)

As you can see in the video, the browser gives an HTTPS warning, [which is probably related to a separate issue](https://github.com/JulianPrieber/llc-docker/issues/53). I'm deploying to [Fly.io](https://fly.io/), which terminates TLS/SSL at their proxy and sends the request to the container so I would have thought that not to be an issue, but at this point I don't know. Maybe an issue with a `X-Forwarded-Proto` header?

This loop could be related to the mount issue or the HTTPS issue; I'm unsure which.

**Comment by ArslanTu** [comment]
Encountered with the same problem. This may work for you too.

1. Finish your initial setup via ip:port
2. Set "Force https" in the config page. (still via ip:port)

Now you can visit it via domain.

**Comment by enthuzy** [comment]
This happened to me after 3.5 update and even on the latest 3.6. I have force ssl enabled. Versions before 3.5 work just fine. Previous discussions pointed to a server configuration, but after reviewing all of the requirements there is nothing further to troubleshoot. No error messages or logs - only redirect.

Was anything else uncovered? I'd love to update. I even tried installing a separate instance on my server for development/testing, but the same loop.

**Comment by enthuzy** [comment]
any endpoint (including /skip) loops back to the default page and removes the trailing page from the url

**Comment by enthuzy** [comment]
Unfortunately, I'm not able to use a docker container with my VPS. What is interesting is that every release prior to 3.5 works just fine on this server. I first updated via the update mechanism which broke the instance. I then tried installing in a separate folder by just unzipping the distributive on the server with it's own domain. Always loops to default.

I also just manually updated the .env file and removed "INSTALLING" file to try a manual setup and still loops to default. I guess what is the major difference between versions prior to 3.5 and now that are causing this loop?
